src/index/views.rs

```diff
@@ -1,4 +1,19 @@
 use crate::prelude::*;
+use pgrx::error;
+use pgrx::pg_sys::{AsPgCStr, RelnameGetRelid};
+
+#[pgrx::pg_extern(volatile, strict)]
+fn _vectors_index_setting(index: String, key: String, value: String) {
```

Comment:
Let's use consistent name.
```suggestion
fn _vectors_alter_vector_index(index: String, key: String, value: String) {
```